packages/lib/modules/pool/PoolDetail/PoolHeader/PoolMetaBadges.tsx

```diff
@@ -46,6 +50,13 @@ export default function PoolMetaBadges() {
       <BalBadge color="font.secondary" fontSize="xs" textTransform="none">
         {getPoolTypeLabel(pool.type)}
       </BalBadge>
+      {hasHook && (
+        <BalBadge color="font.primary" fontSize="xs" h={8} w={8}>
```

Comment:
I would add a popover like we have in the swap fee icon: 

<img width="393" alt="hookTooltip" src="https://github.com/user-attachments/assets/7cd7a1e9-760b-4288-a599-dda5f519c1a4">

If it's easy we could show a link to the name of the hook or a link to the hook contract in the popover. (can be done in the future)

Reply:
Yeah metadata is coming in another PR.